editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -1652,10 +1629,6 @@ impl Fsm for PathToolFsmState {
 					HintInfo::keys([Key::Alt], "Subtract").prepend_plus(),
 				]),
 			]),
```

Comment:
You are missing a hint for the new <kbd>Ctrl</kbd> to delete segment behaviour I think?